src/sentry/models/docintegrationavatar.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+from sentry.db.models import FlexibleForeignKey
+
+from . import AvatarBase
+
+
+class DocIntegrationAvatar(AvatarBase):
+    """
+    A DocIntegrationAvatar associates a DocIntegration with
+        a logo photo File.
+    """
+
+    FILE_TYPE = "avatar.file"
+
+    doc_integration = FlexibleForeignKey("sentry.DocIntegration", related_name="avatar")
+    avatar_type = models.PositiveSmallIntegerField(default=0, choices=((0, "upload"),))
+
```

Comment:
Could be helpful to have `date_added`. Could potentially just use `DefaultFieldsModel` then we get both updated/added?

Reply:
Cool, I added a `date_created` field - I ran into an issue trying to inherit from both `AvatarBase` (which we need) and `DefaultFieldsModel` because there were 2 `AutoFields`, so I just manually added one.